frontend/app/assets/javascripts/spree/frontend/checkout/address.js

```diff
@@ -27,6 +29,12 @@ Spree.ready(function ($) {
         }
       }
 
+      Spree.addTranslation = function () {
+        Spree.translations = {
+          country_state: Spree.t('state')
```

Comment:
can't we just add this translation in the translations.html.erb? not sure this function is necessary.

Spree.translations = <%==
      {
        coupon_code_applied: Spree.t(:coupon_code_applied),
        coupon_code_removed: Spree.t(:coupon_code_removed),
        coupon_code_error_icon: image_path('error.svg'),
        country_state: Spree.t('state')
      }.to_json
    %>

Reply:
If I add this code there it won't be available when I js code but after it.